WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderSearchActivity.kt

```diff
@@ -41,4 +47,23 @@ class ReaderSearchActivity : LocaleAwareActivity() {
         super.onPause()
         readerTracker.stop(MAIN_READER)
     }
+
+    override fun onScrollableViewInitialized(containerId: Int) {
+        val fragmentContainer = supportFragmentManager.findFragmentById(R.id.fragment_container)
+
+        if (fragmentContainer is ReaderPostListFragment) {
+            val fragmentView = fragmentContainer.view ?: return
+
+            fragmentView.post {
+                // post is used to create a minimal delay here. containerId changes just before
+                // onScrollableViewInitialized is called, and findViewById can't find the new id before the delay.
+                val jetpackBannerView = fragmentView.findViewById<View>(R.id.jetpack_banner)
+                val scrollableView = fragmentView.findViewById<View>(containerId) as RecyclerView
+
+                if (jetpackBrandingUtils.shouldShowJetpackBranding()) {
+                    jetpackBrandingUtils.initJetpackBannerAnimation(jetpackBannerView, scrollableView)
+                }
+            }
+        }
+    }
 }
```

Comment:
I couldn't understand why we inform `ReaderSearchActivity` from `ReaderPostListFragment` about `onScrollableViewInitialized`. We don't do anything related to `ReaderSearchActivity` here, `jetpackBannerView` and `scrollableView` are both in `ReaderPostListFragment`. Can we move this to `ReaderPostListFragment`? 🤔

Reply:
I saw other activities that implement `ScrollableViewInitializedListener` using a similar approach, and I went after doing the same.

By getting rid of `ScrollableViewInitializedListener` this should work directly from the fragment 👍

Reply:
Done in https://github.com/wordpress-mobile/WordPress-Android/pull/17028/commits/159f018cbc149575214d3727501dfc994e62150e